maps/antag_spawn/ert/ert.dm

```diff
@@ -42,18 +42,59 @@
 
 /area/map_template/rescue_base
```

Comment:
Added a bunch of areas to the ERT Base so it actually works even though it doesn't matter since none of them use power.